### PR TITLE
Changes required for project creator

### PR DESCRIPTION
**Pull request by K-Meech**
Changes needed for this PR: https://github.com/mobie/mobie-viewer-fiji/pull/426

Changes:
- changes line in 'XmlN5OmeZarrImageLoader' to allow it to work on Windows (on windows the separator is \ rather than /).
- adds toXml() method for XmlN5S3OmeZarrImageLoader needed for writing remote metadata (same structure as https://github.com/mobie/bigdataviewer-imageloader/blob/cacefde8e46725647eac5524d03c83302cdc0608/src/main/java/de/embl/cba/n5/util/loaders/xml/XmlIoN5S3ImageLoader.java#L58)

### CODE REVIEW

**Review by KateMoreva — APPROVED** [review]

**Comment by K-Meech** [comment]
@KateMoreva - can you merge this PR, if you're happy with it? Also, could this go on the version on the maven repo? (we'll need it for building mobie once I merge: mobie/mobie-viewer-fiji#426) Thanks!

**Comment by KateMoreva** [comment]
> @KateMoreva - can you merge this PR, if you're happy with it? Also, could this go on the version on the maven repo? (we'll need it for building mobie once I merge: [mobie/mobie-viewer-fiji#426](https://github.com/mobie/mobie-viewer-fiji/pull/426)) Thanks!

@K-Meech, files in the maven repo were updated. Please, make sure that you have fresh files.

**Comment by K-Meech** [comment]
Perfect - thanks @KateMoreva!

**Pull request merged**
